scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -354,3 +376,37 @@ task SumBytes {
     cpu: 1
   }
 }
+
+task CreateManifest {
+
+    input {
+        Array[String] manifest_lines
```

Comment:
are these ordered? If so, I suppose this task could also be responsible for numbering the rows.

Reply:
yeah. I wanted them to be ordered which is why I run it through `sort -n`. It's also why I have the index as the first column.. I can see if I can try something smarter by using a Map in WDL.

Reply:
I couldn't find a way to make this work with the built in WDL functions.. I can package the interval_index and manifest line with their `Pair` object but I don't see a reasonable way to actually print the values out into the command block. Given that, I'm going to pass the index through as part of the manifest line like I was doing before but I am at least passing IN the index as part of the scatter call instead of parsing it from the filename.